tests/unit/scala/test_error_message.py

```diff
@@ -380,6 +380,14 @@ def test_server_udf_upload_file_stream_closed():
     assert "A file stream was closed when uploading UDF files" in ex.message
 
 
+def test_server_upload_file_stream_closed():
+    dest_filename = "file"
+    ex = SnowparkClientExceptionMessages.SERVER_UPLOAD_FILE_STREAM_CLOSED(dest_filename)
+    assert isinstance(ex, SnowparkUploadUdfFileException)
```

Comment:
oops, I just noticed this error. Line 386 need to check for instance `SnowparkUploadUdfFileException`